src/main/scala/strawman/collection/mutable/ArrayBuffer.scala

```diff
@@ -39,7 +39,7 @@ class ArrayBuffer[A] private (initElems: Array[AnyRef], initLength: Int)
         start = 0
       }
       else {
-        val newelems = new Array[AnyRef](end * 2)
+        val newelems = new Array[AnyRef](if (end == 0) 16 else (end * 2))
```

Comment:
I added this because there was an issue in the implementation if the initial size was 0.